.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Check
```

Comment:
🛑 Shouldn't this be renamed to `Publish`?

Reply:
Oh, thanks!